pkg/runtime/k0s/k0sctl/v1beta1/types.go

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
```

Comment:
If this is the code of the k0s project, please keep the k0s license. Or could we use those struct directly ?

Reply:
Let me keep original license.